docs/pages/foundations/typography/guidelines.js

```diff
@@ -7,6 +7,17 @@ import Markdown from '../../../docs-components/Markdown.js';
 import Page from '../../../docs-components/Page.js';
 import PageHeader from '../../../docs-components/PageHeader.js';
 import PrincipleItem from '../../../docs-components/PrincipleItem.js';
+import SandpackExample from '../../../docs-components/SandpackExample.js';
+import alignAndSpaceTextForReadability from '../../../examples/iconography/alignAndSpaceTextForReadability.js';
+import dontCenterTextWithTightLeading from '../../../examples/iconography/dontCenterTextWithTightLeading.js';
+import dontTruncateParagraphText from '../../../examples/iconography/dontTruncateParagraphText.js';
```

Comment:
should we move this examples to examples/typography?

Reply:
yes